src/SelfService/Application/AwsEcrRepositoryApplicationService.cs

```diff
@@ -53,20 +53,20 @@ public async Task CreateECRRepo(string repositoryName)
             new CreateRepositoryRequest
             {
                 ImageScanningConfiguration = new ImageScanningConfiguration() { ScanOnPush = true },
-                RepositoryName = repositoryName,
+                RepositoryName = name,
```

Comment:
guess AWS is calling is RepositoryName 😂

Reply:
Yes, that was always the case